zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/EncryptionUtils.java

```diff
@@ -64,4 +68,16 @@ public static InputStream encryptionInputStream(InputStream inputStream, SecretK
     public static InputStream encryptionInputStream(Path path, SecretKey key) throws IOException {
         return encryptionInputStream(Files.newInputStream(path), key);
     }
+
+    public static String createMD5HashSum(String value) {
+        try {
+            byte[] bytesOfMessage = value.getBytes("UTF-8");
+            MessageDigest md = MessageDigest.getInstance("MD5");
+            byte[] theMD5digest = md.digest(bytesOfMessage);
+            return  DatatypeConverter.printHexBinary(theMD5digest).toUpperCase();
```

Comment:
Two spaces after `return`.

But more importantly, the API only accepts lowercase md5 hashes as IDs, and you don't seem to be setting this value to lowercase later on (or at least I couldn't find it), so wouldn't this end up generating a 400 error in the API?

Reply:
have changed this to lower case, and we plan to test this locally to ensure it all works as expected